modules/feature-management/src/Volo.Abp.FeatureManagement.Blazor/Components/FeatureManagementModal.razor

```diff
@@ -6,15 +6,15 @@
     <ModalContent Size="ModalSize.Large" Centered="true">
         <ModalHeader>
             <ModalTitle>@L["Features"]</ModalTitle>
-            <CloseButton Clicked="CloseModal"/>
+            <CloseButton Clicked="CloseModal" />
         </ModalHeader>
-        <ModalBody MaxHeight="50">
-            @if (Groups == null || !Groups.Any())
-            {
-                <span>@L["NoFeatureFoundMessage"]</span>
-            }
-            else
-            {
+        @if (Groups == null || !Groups.Any())
+        {
+            <span>@L["NoFeatureFoundMessage"]</span>
```

Comment:
@malikmasis thanks for the Empty State. But we can use a little space around the text I think
```suggestion
            <span class="m-3">@L["NoFeatureFoundMessage"]</span>
```